tests/test_taxi_service_forms.py

```diff
@@ -0,0 +1,89 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Car, Manufacturer
+
+TestCase.fixtures = ["taxi_service_db_data.json", ]
```

Comment:
For this task, fixtures are not the best option.
You don't need a lot of testing data to check if it is working or not.
You do not test pagination, you do not test listing of instances.
You check create/update/delete, which should be done without fixtures, just with creating data during the tests

Reply:
As soon as you do not test pagination, you do not test listing of instances. You only check create/update/delete, which should be done without fixtures, just with creating data during the tests
This will be more precise.